lib/Messenger/Handler/AssetUpdateTasksHandler.php

```diff
@@ -62,6 +62,10 @@ private function saveAsset(Asset $asset): void
 
     private function processDocument(Asset\Document $asset): void
     {
+        if ($asset->getMimeType() === 'application/pdf' && $asset->checkIfPdfContainsJS()) {
```

Comment:
We need to check the config (enabled/disabled) here as well, right?

Reply:
this happens in `checkIfPdfContainsJS` https://github.com/pimcore/pimcore/blob/a8b048d8ec5a13cfcf390f767f9859442ae1b3ca/models/Asset/Document.php#L170